zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -396,6 +396,11 @@ table.table-shortcut {
  opacity: 0 !important;
 }
 
+#main .emacs-mode .ace_cursor {
+  background: #C0C0C0!important;
```

Comment:
@malayhm Thanks for the contribution.

Can we apply normal thin cursor style of ace editor? This is style for that, (you can find by searching `ace_cursor`css class in chrome developer tool as well)

![image](https://user-images.githubusercontent.com/4968473/27845518-07e1e18e-616b-11e7-8ec2-cfa9ada00037.png)

Thick cursor comes from emacs mode which enabled only in OSX. In other words, other platforms already use thin cursor provided by ace editor.

```js
// paragraph.controller.js #L 690~

      if (navigator.appVersion.indexOf('Mac') !== -1) {
        $scope.editor.setKeyboardHandler('ace/keyboard/emacs')
        $rootScope.isMac = true
      } else if (navigator.appVersion.indexOf('Win') !== -1 ||
        navigator.appVersion.indexOf('X11') !== -1 ||
        navigator.appVersion.indexOf('Linux') !== -1) {
        $rootScope.isMac = false
        // not applying emacs key binding while the binding override Ctrl-v. default behavior of paste text on windows.
      }
```